app/views/events/_event.html.erb

```diff
@@ -1,7 +1,7 @@
-<div>
-  <div style="margin-bottom: 5px"><b>Event title:</b> <%= event.title %></div>
-  <div style="margin-bottom: 5px"><b>Event description:</b> <%= event.description %></div>
-  <div style="margin-bottom: 5px"><b>Event date:</b> <%= event.date %></div>
-  <div style="margin-bottom: 5px"><b>Event location:</b> <%= event.location %></div>
-  <div style="margin-bottom: 15px"><b>Event price:</b> <%= event.price %></div>
+<div class= "grid grid-flow-row gap-3">
+  <div><b class="b">Event title:</b> <%= event.title%></div>
+  <div><b class="b">Event description:</b> <%= event.description %></div>
+  <div><b class="b">Event date:</b> <%= event.date %></div>
+  <div><b class="b">Event location:</b> <%= event.location %></div>
+  <div><b class="b">Event price:</b> <%= event.price %></div>
```

Comment:
If you are using tailwind, why don't you apply the font styles directly in the <b></b> tags?

Reply:
Fixed!